checker/src/org/checkerframework/checker/lock/LockTransfer.java

```diff
@@ -173,4 +196,18 @@ else if (astKind == Kind.ARBITRARY_CODE) { // Handle initializers
 
         return result;
     }
+
+    @Override
+    public TransferResult<CFValue, LockStore> visitStringConcatenate(
```

Comment:
I removed this code and added handling of string concatenation to LockTreeAnnotator.